--- dchub-0.5.2/configure.in~	2013-06-09 20:37:05.851754198 +0200
+++ dchub-0.5.2/configure.in	2013-06-09 20:37:30.785086651 +0200
@@ -3,7 +3,7 @@
 
 AC_INIT(configure.in)
 AM_INIT_AUTOMAKE(dchub, 0.5.2)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 AM_SANITY_CHECK
 AC_ISC_POSIX
@@ -13,7 +13,6 @@
 AC_PROG_CC
 AC_PROG_INSTALL
 AC_PROG_LN_S
-AM_PROG_CC_STDC
 
 dnl dnl check for msgfmt
 AC_CHECK_PROG(HAVE_MSGFMT,msgfmt,yes,no)
