diff -urNp -x '*.orig' dchub-0.5.2.org/Makefile.am dchub-0.5.2/Makefile.am
--- dchub-0.5.2.org/Makefile.am	2003-10-11 13:58:24.000000000 +0200
+++ dchub-0.5.2/Makefile.am	2021-05-07 22:56:59.471651622 +0200
@@ -11,7 +11,8 @@ edit = sed \
 	-e 's,@libexecdir\@,$(libexecdir),g' \
 	-e 's,@VERSION\@,$(VERSION),g' \
 	-e 's,@INITDIR\@,$(INITDIR),g' \
-	-e 's,@DCHUB_DBDIR\@,$(DCHUB_DBDIR),g'
+	-e 's,@DCHUB_DBDIR\@,$(DCHUB_DBDIR),g' \
+	-e 's,@DCHUB_CRCDIR\@,$(DCHUB_CRCDIR),g'
 
 dchub.spec: Makefile
 	$(edit) $(srcdir)/dchub.spec.in >dchub.spec
diff -urNp -x '*.orig' dchub-0.5.2.org/src/Makefile.am dchub-0.5.2/src/Makefile.am
--- dchub-0.5.2.org/src/Makefile.am	2003-11-28 15:39:31.000000000 +0100
+++ dchub-0.5.2/src/Makefile.am	2021-05-07 22:56:59.471651622 +0200
@@ -1,7 +1,7 @@
 # $Id: Makefile.am,v 2.44 2003/11/28 14:39:31 ericprev Exp $
 INCLUDES = $(GLIB_CFLAGS) $(XML_CFLAGS) $(perl_cflags) -DWITH_PERL
 
-CPPFLAGS = -DDCHUB_DBDIR=\"$(DCHUB_DBDIR)\" -DDCHUB_TEMPLATEDIR=\"$(DCHUB_TEMPLATEDIR)\" @CPPFLAGS@
+CPPFLAGS = -DDCHUB_CRCDIR=\"$(DCHUB_CRCDIR)\" -DDCHUB_DBDIR=\"$(DCHUB_DBDIR)\" -DDCHUB_TEMPLATEDIR=\"$(DCHUB_TEMPLATEDIR)\" @CPPFLAGS@
 
 DISTCLEANFILES = *~
 EXTRA_DIST = inet_aton.c FILE_DESCRIPTION dchub.dtd users.xml.in conf.xml.in
diff -urNp -x '*.orig' dchub-0.5.2.org/src/main.c dchub-0.5.2/src/main.c
--- dchub-0.5.2.org/src/main.c	2003-11-22 14:47:08.000000000 +0100
+++ dchub-0.5.2/src/main.c	2021-05-07 22:56:59.471651622 +0200
@@ -1224,7 +1224,7 @@ static void parse_command_line_options(i
 	if(ed2kdb_file==NULL)
 	{
 		GString* default_ed2kdb_filename=g_string_new("");
-		g_string_sprintf(default_ed2kdb_filename,"%s/%s",DCHUB_DBDIR,"dchub.crc");
+		g_string_sprintf(default_ed2kdb_filename,"%s/%s",DCHUB_CRCDIR,"dchub.crc");
 		ed2kdb_file=strdup(default_ed2kdb_filename->str);
 		g_string_free(default_ed2kdb_filename, TRUE);
 	}
